cmd/ipsec/ipsec-stats.go

```diff
@@ -12,23 +12,27 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// AddrCmdLineArg cmdline arg name for address
+const AddrCmdLineArg = "addr"
+
+// TLSFiles cmdline arg name for tls files
+const TLSFiles = "tlsfiles"
```

Comment:
is it possible to move these addr and tls constants in some common location in order not to repeat it again and again?